src/settings.py

```diff
@@ -32,231 +32,248 @@
 MANAGERS = ADMINS
 
 DATABASES = {
-    'default': {
-        'ENGINE': 'djongo',
-        'NAME': 'wstore_db',
-        'ENFORCE_SCHEMA': False,
-        'CLIENT': {
-            'host': 'localhost',
+    "default": {
+        "ENGINE": "djongo",
+        "NAME": "wstore_db",
+        "ENFORCE_SCHEMA": False,
+        "CLIENT": {
+            "host": "charging_mongo",
```

Comment:
You need to change this one to localhost again

Reply:
Sorry for the mistake, fixed it in new commmit.